website/frontend/management/commands/scraper.py

```diff
@@ -265,12 +292,15 @@ def get_hash(version):
                 output = run_git_command(['ls-tree', '-r', version, filename],
                                          article.full_git_dir)
                 return output.split()[2]
+            # What's the difference between `hashes` and `commits`?
+            # `hashes` is the file hash, `commits` are the commit hashes
             hashes = map(get_hash, commits)
 
             number_equal = sum(1 for h in hashes if h == my_hash)
 
             logger.debug('Got %s previous version files have an identical hash', number_equal)
 
+            # So if the version has reverted to a previous version, the system might not show it...
```

Comment:
Yeah.  But at least if it bounces back and forth and settles on the new version, it will show that.